libraries/AC_PID/AC_P_1D.h

```diff
@@ -17,22 +17,29 @@ class AC_P_1D {
     // set time step in seconds
     void set_dt(float dt) { _dt = dt; }
 
+    // get maximum error
+    float get_error_min() const { return _error_min; }
+    float get_error_max() const { return _error_max; }
+
     // update_all - set target and measured inputs to P controller and calculate outputs
     // target and measurement are filtered
-    // if measurement is further than error_min or error_max (see set_limits_error method)
+    // if measurement is further than error_min or error_max (see set_limits method)
     //   the target is moved closer to the measurement and limit_min or limit_max will be set true
     float update_all(float &target, float measurement, bool &limit_min, bool &limit_max) WARN_IF_UNUSED;
 
+    // set limits on error, output and output from D term
+    void set_limits(float error_min, float error_max, float output_min, float output_max, float D1_max = 0.0f, float D2_max = 0.0f);
+    float get_error_min() { return _error_min; }
```

Comment:
Looks like we have duplicate calls called get_error_min() and get_error_max().  This is possible because one of them is "const" and one is not.  We should remove this one which isn't const.

Reply:
That hit my blind spot! I had to do a search to find the other ones and it was only a couple lines above!